### DISCUSSION TITLE
Getting more info about failed installs and downloads

### DISCUSSION THREAD

**Discussion by schneidemar** [yarnpkg/berry#3086] [Q&A] [answered]
I'm using an Github Action workflow to build the project on every master branch push. While `yarn` does work locally, the installation does not work in the workflow. I'm seeing following logs:
```
Run yarn
➤ YN0000: ┌ Resolution step
Resolution step
  ➤ YN0001: │ HTTPError: Response code 404 (Not Found)
      at se.<anonymous> (/home/runner/work/lvl1-helpers/lvl1-helpers/.yarn/releases/yarn-2.4.2.cjs:23:10082)
      at runMicrotasks (<anonymous>)
      at processTicksAndRejections (internal/process/task_queues.js:95:5)
```

The 'HTTPError' is not very helpful in tackling down the error. How can I gain more insight? There is no `--verbose` or the like.
Any help (also for local execution) appreciated, thanks in advance!

**Comment by merceyz** [comment, marked as answer]
The http errors were improved in https://github.com/yarnpkg/berry/pull/2707 (see examples here https://github.com/yarnpkg/berry/pull/2406) and released in `3.0.0-rc.1`

You can get the latest RC (at the time of writing) by running
```
yarn set version 3.0.0-rc.7
```

As for why you're getting the error I'm guessing you're not authenticated

**Reply by schneidemar** [reply]
Thanks, that gives useful info about the error.

Exactly, there's an issue with authentication, and now I know where, so I can investigate further.
